cmd/thv-proxyrunner/main.go

```diff
@@ -0,0 +1,25 @@
+// Package main is the entry point for the ToolHive CLI.
+package main
+
+import (
+	"os"
+
+	"github.com/stacklok/toolhive/cmd/thv-proxyrunner/app"
+	"github.com/stacklok/toolhive/pkg/kubernetes/client"
+	"github.com/stacklok/toolhive/pkg/kubernetes/container"
+	"github.com/stacklok/toolhive/pkg/kubernetes/logger"
+)
+
+func main() {
+	// Initialize the logger
+	logger.Initialize()
+
+	// Check and perform auto-discovery migration if needed
+	// Handles the auto-discovery flag depreciation, only executes once on old config files
+	client.CheckAndPerformAutoDiscoveryMigration()
+
+	// Skip update check for completion command or if we are running in kubernetes
+	if err := app.NewRootCmd(!app.IsCompletionCommand(os.Args) && !container.IsKubernetesRuntime()).Execute(); err != nil {
```

Comment:
Since we are always in k8s, we can get rid of this check, right?

Reply:
Yeah I was going to get rid of it in another PR, I wanted to just do a straight copy/paste just to reduce the need for folks to go searching for any custom changes. Am happy to do it in this PR though if you want.